x-pack/metricbeat/module/aws/cloudwatch/cloudwatch.go

```diff
@@ -139,12 +139,16 @@ func (m *MetricSet) Fetch(report mb.ReporterV2) error {
 	}
 
 	// Create events based on namespaceDetailTotal from configuration
```

Comment:
Is this comment still valid after the switch in the loops?

Reply:
Yeah it is still valid but definitely at the wrong spot. I will move it to the next for loop. Thanks!